lib/shoryuken/middleware/server/exponential_backoff_retry.rb

```diff
@@ -12,14 +12,19 @@ def call(worker, queue, sqs_msg, body)
 
           started_at = Time.now
           yield
-        rescue
+        rescue => ex
+          logger.info { "Message #{sqs_msg.message_id} will attempt retry due to error: #{ex.message}" }
```

Comment:
@rwadstein I'm wondering if that should be below line 24, since 23 can `raise` up the error, WDYT?

Should this be also `debug`?